apps/store/src/components/CartInventory/CartInventory.tsx

```diff
@@ -20,17 +27,57 @@ export const CartInventory = ({ cart, children }: Props) => {
         ))}
       </List>
       <Footer>
-        <Heading as="h3" variant="standard.18">
-          Total
-        </Heading>
-        <Heading as="h3" variant="standard.18">
-          {formatter.monthlyPrice(cart.cost.net)}
-        </Heading>
+        <TotalWrapper>
+          <Heading as="h3" variant="standard.18">
+            {t('CHECKOUT_PRICE_TOTAL')}
+          </Heading>
+          <DisplayTotalAmount cost={cart.cost} hasDiscount={hasDiscount} />
+        </TotalWrapper>
+        {campaigns
+          ? campaigns.map((campaign) => (
+              <DiscountMessage key={campaign.discount.type}>
+                {discountDurationExplanation(campaign.discount, cart.cost.gross)}
+              </DiscountMessage>
+            ))
+          : null}
       </Footer>
     </Space>
   )
 }
 
+type DisplayTotalAmountProps = {
+  cost: CartFragmentFragment['cost']
+  hasDiscount: boolean
```

Comment:
very nit: Since we're defining a a default value for `hasDiscount` maybe it should be optional.

```suggestion
  hasDiscount?: boolean
```

Reply:
I've got a little habit of adding default values. 

Here I decided to remove the default value and keep `hasDiscount` as a required prop.